dotnet-authserver/src/TeacherIdentity.AuthServer/Oidc/CustomClaims.cs

```diff
@@ -9,4 +9,5 @@ public static class CustomClaims
     public const string HaveCompletedTrnLookup = "completed-trn-lookup";
     public const string UserType = "user-type";
     public const string PreviousUserId = "previous-user-id";
+    public const string PreferredName = "preferred-name";
```

Comment:
Used a `-` to separate words for consistency even though `_` is used in 'official' claims. Changing the claims above now would be a breaking change.